backend/inspirehep/records/marshmallow/literature/bibtex.py

```diff
@@ -54,13 +54,15 @@ class BibTexCommonSchema(Schema):
         "book": "book",
         "book chapter": "inbook",
         "proceedings": "proceedings",
-        "report": "techreport",
+        "report": lambda data: "article"
+        if get_value(data, "publication_info.journal_title")
+        else "techreport",
```

Comment:
sorry, change of plans. Actually `techreport` doesn't make sense as it requires an `institution` field which we don't have. Let's make it `article` always instead.

Reply:
No worries, changed